_fs/root-skel/etc/rc.conf.d/network

```diff
@@ -2,11 +2,15 @@
 # Network configuration
 #
 
-network_interfaces="en1"
+network_interfaces="en1 en2"
```

Comment:
maybe it would be better to put this config in project-specific rootfs-overlay (`_projects/armv7a7-imx6ull-evk/rootfs-overlay/`) as other projects might have only one interface (it seems like a sane default ;))